src/DonationForms/FormDesigns/ClassicFormDesign/js/main.ts

```diff
@@ -1,5 +1,4 @@
-import Goal from './layouts/Goal';
 import Header from './layouts/Header';
 
-window.givewp.form.templates.layouts.goal = Goal;
+// window.givewp.form.templates.layouts.goal = Goal;
```

Comment:
Does this commented0out line need to be removed?

Reply:
Good catch. Resolved in 9b84d74dc